src/tic.c

```diff
@@ -197,7 +198,7 @@ static void setPixel(tic_machine* machine, s32 x, s32 y, u8 color)
 {
 	if(x < machine->state.clip.l || y < machine->state.clip.t || x >= machine->state.clip.r || y >= machine->state.clip.b) return;
 
-	machine->state.setpix(&machine->memory, x, y, color);
+	machine->state.setpix(&machine->memory, x, y, mapColor(&machine->memory, color));
```

Comment:
I moved the mapColor here.

Reply:
ahh, ok, I didn't notice this, sorry :)
merging...